src/actions/index.js

```diff
@@ -1,3 +1,4 @@
 export * from './pageNumActions';
 export * from './exportCSV'
+export * from './exportPDF'
```

Comment:
ah would really like these "actions" put somewhere else now that they're implemented.